README.md

```diff
@@ -78,6 +78,10 @@ provider "pulsar" {
 | `tls_trust_certs_file_path`     | Path to a custom trusted TLS certificate file                                                                          | No       |
 | `tls_allow_insecure_connection` | Boolean flag to accept untrusted TLS certificates                                                                      | No       |
 | `api_version`                   | Used to request Apache Pulsar API service, default by 0, which represents use default version                          | No       |      
+| `audience`                      | The OAuth 2.0 resource server identifier for the Pulsar cluster                                                        | No       |  
```

Comment:
Maybe we need to consider adding another field `scope` in order to support azure AD OAuth2 https://github.com/streamnative/pulsarctl/pull/465